### PR TITLE
 Camel-case filename in uploadFile.

### LINKED ISSUE #43: filename vs fileName inconsistency
The method `b2.uploadFile()` seems to use `filename` as an arg, while most (if not all) others like `b2.deleteFileVersion()` use `fileName`.

Perhaps we could change `b2.uploadFile()` to use `fileName` by default, but fallback to `filename` so we don't break the library for everyone?

### PR DESCRIPTION
**Pull request by odensc**
Fixes #43.

### CODE REVIEW

File: lib/actions/file.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ exports.uploadFile = function(b2, args) {
     var uploadUrl = args.uploadUrl;
     var uploadAuthToken = args.uploadAuthToken;
     // Previous versions used filename (lowercase), so support that here
-    var fileName = utils.getUrlEncodedFileName(typeof args.fileName !== "undefined" ? args.fileName : args.filename);
+    var fileName = utils.getUrlEncodedFileName(typeof args.fileName !== 'undefined' ? args.fileName : args.filename);
```
Comment (cbess) [inline lib/actions/file.js]: I'd say we can reduce it to:
```js
var fileName = utils.getUrlEncodedFileName(args.fileName || args.filename);
```

Comment (odensc) [reply lib/actions/file.js]: It was originally that but I changed it to a `typeof` check because the `||` wouldn't work if `fileName` was `'0'`.

Comment (cbess) [reply lib/actions/file.js]: agreed, if someone uses an integer we have a different problem :) I prefer the brevity

Comment (cbess) [reply lib/actions/file.js]: Also, the string `'0'` would be chosen:
`'0' || true == '0'`

Comment (odensc) [reply lib/actions/file.js]: Ah interesting, I always thought it was casted into `0` in that situation. The more you know! I reverted back and squashed into one commit.

**Comment by cbess** [comment]
Looks good to me

**Pull request merged**